src/pages/connectionPage/ui/ConnectionPage.tsx

```diff
@@ -0,0 +1,29 @@
+import {TonConnectButton, useTonAddress} from "@tonconnect/ui-react";
+import "@twa-dev/sdk";
+import css from './ConnectionPage.module.css'
+
+export function ConnectionPage() {
+    const userFriendlyAddress = useTonAddress();
+
+    const sendWalletAddressHandler = () => {
+        if (userFriendlyAddress) window.Telegram.WebApp.sendData(userFriendlyAddress)
+    }
+
+    return (
+        <>
+            {userFriendlyAddress ?
+                (<div className={css.root}>
+                    <h1>Четко. Вы готовы закупаться!</h1>
+                    <button className={css.startSellButton}
+                            onClick={sendWalletAddressHandler}
```

Comment:
Not sure if we should close app and send address by **this** button.
I thought that on the first login front would send address right after login (without any additional user action). And all the subsequent login page openings will show this button, that will lead to some another page